docs/running_inclavare_containers_with_pouch_and_occlum.md

```diff
@@ -17,14 +17,21 @@ Please refer to [this guide](https://github.com/alibaba/inclavare-containers/tre
 
 # Running Occlum container image
 
-Please refer to [this guide](https://github.com/occlum/occlum/blob/master/docs/rune_quick_start.md) to build your Occlum container image. Then run pouch with Occlum container images refer to
+Please refer to [this guide](https://github.com/occlum/occlum/blob/master/docs/rune_quick_start.md) to build your Occlum container image. 
+
+You can use `OCCLUM_RELEASE_ENCLAVE` env to set your enclave type:
```

Comment:
change to: use the environment variable `OCCLUM_RELEASE_ENCLAVE` to set your enclave type:

Reply:
> change to: use the environment variable `OCCLUM_RELEASE_ENCLAVE` to set your enclave type:

Done